README.md

```diff
@@ -26,6 +26,50 @@ Prerequisites:
 Installation (make)
 ------------
 
+### Installing ls1-MarDyn using cmake
+This is the recommended way of building ls1-MarDyn.
+
+#### Configuration
+To build ls1-MarDyn using cmake first create an additional directory on the root ls1-MarDyn directory and change into that directory.
+```bash
+mkdir build
+cd build
+```
+Next, `cmake` has to be executed. In most cases, you will have to specify the compiler with which ls1-MarDyn should be built:
+```bash
+CC=clang CXX=clang++ cmake ..
+```
+Some of the most common compilers are
+| Compiler      | CC    | CXX     |
+|---------------|-------|---------|
+| GCC           | `gcc`   | `g++`     |
+| Intel Classic | `icc`   | `icpc`    |
+| Intel oneAPI  | `icx`   | `icpx`    |
+| Clang         | `clang` | `clang++` |
+
+Specifying the compiler is only possible at the first execution of cmake.
```

Comment:
```suggestion
Specifying the compiler this way is only possible at the first execution of cmake.
```
It is possible to change it afterwards but you have to know what to delete so starting over is easier.